src/react/experimental/viewport-transition-utils.js

```diff
@@ -17,6 +37,23 @@ export function extractViewportFrom(props) {
   return viewport;
 }
 
+/* eslint-disable max-depth */
+export function areViewportsEqual(startViewport, endViewport) {
+  for (const p of VIEWPORT_INTERPOLATION_PROPS) {
```

Comment:
This will skip viewport updates when only non interpolation props (width or height) changed. `VIEWPORT_INTERPOLATION_PROPS` should be replaced with `VIEWPORT_PROPS`.

Reply:
This is intentional We do not want to trigger transition when viewport size changes.

Reply:
Yes, but we we ignoring width/height changes during transitions and map is not resized.

Reply:
Addressed in https://github.com/uber/deck.gl/pull/1063/commits/a6b4600ec7fcccd45639b9b51f3fabe91255435c